app/helpers/movie_helper.rb

```diff
@@ -0,0 +1,14 @@
+module MovieHelper
+  IMDB_MOVIE_BASE_URL = 'https://www.imdb.com/title'.freeze
+  TMDB_MOVIE_BASE_URL = 'https://www.themoviedb.org/movie'.freeze
+
+  def imdb_url(movie)
+    return if movie.imdb_id.nil?
```

Comment:
This function (and the one below) should receive what they need to do what they are doing: interpolating a string (constant) they know — the base URLs — in a format that they also know with an ID they receive.

Why do they get a `movie`? For one, there may be other objects that have a movie ID beside the one that `respond_to?(:imdb_id)`, and also this is coupled with the knowledge that `imdb_id` can be blank/`nil` for `imdb_id` but not for `tmdb_id` when it shouldn't.